src/js/api/api.js

```diff
@@ -494,7 +501,12 @@ define([
                 }
             });
 
-            _this.trigger(events.API_READY, {});
+            _qoe.tick(events.API_READY);
+
+            _this.trigger(events.API_READY, {
+                initTime  : _qoe.between(events.API_INITIALIZED, events.API_SETUP),
```

Comment:
"initTime" will confuse users and is not relevant to QoE. For example I can init the api ```var jw = jwplayer()``` and then wait before calling ```jw.setup()```. We may also choose to call setup multiple times on the same api instance.

I like that initialization time is tracked, but don't see a use for this measurement. I'd rather have a call that gives me a list of sorted events to visualize API lifespan... but that's another story.

What we might want is a library qoe instance that measures script execution on load. I have a feeling that's what users would think is "initTime". This would probably require some addition to webpack.